upload/Uploader.js

```diff
@@ -200,7 +200,7 @@ class Uploader {
                 });
 
                 // Fetch the current gas price and increase it
-                const currentGasPrice = await this.#wallet.provider.getGasPrice();
+                const currentGasPrice = await this.#wallet.getGasPrice();
```

Comment:
There is no gasPrice function anymore, the latest one is `wallet.provider.getFeeData()`.

Reply:
FeeData {
  gasPrice: 1000839477n,
  maxFeePerGas: 1001678954n,
  maxPriorityFeePerGas: 1000000000n
}

Reply:
This problem has been fixed and the PR can be turned off.